src/test/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvertTests.java

```diff
@@ -0,0 +1,124 @@
+package io.github.oliviercailloux.y2018.apartments.utils;
+
+import io.github.oliviercailloux.y2018.apartments.apartment.Apartment;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Test class for JsonConvert
+ * 
+ * @author Etienne CARTIER & Morgane FIOT
+ */
+public class JsonConvertTests {
```

Comment:
La convention veut qu'on ne mette pas de "s" à la fin du nom de la classe

```suggestion
public class JsonConvertTest {
```